README.md

```diff
@@ -99,6 +99,64 @@ app('api.router')->version(config('api.version'), ['namespace' => 'Saritasa\Lara
 });
 ```
 
+## Routes registration examples:
+
+### Simple route
+```php
+$registrar->get('users', ApiController::class, 'list');
+```
+In this case ApiController::list will be calling with default dependency injection.
+
+### Route with parameters
+```php
+
+// Controller exmaple
+class ApiController
+{
+    public function show(int $id)
+    {
+    }
+}
+
+// Route example
+$registrar->get('user/{id}', ApiController::class, 'show');
+```
+In this case ApiController::show will receive directly parameter from url. Ex: /user/5
+
+###Route with binding on controller side
+```php
+// Controller exmaple
+class ApiController
+{
+    public function show(User $user)
+	{
+	}
+}
+
+// Route example
+$registrar->get('user/{user}', ApiController::class, 'show');
```

Comment:
please, use plural forms for resource names even for examples (**users/{user}**)